app/src/main/java/com/example/gfood/utils/ActivityExts.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.gfood.utils
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+
+inline fun FragmentManager.inTransaction(func: FragmentTransaction.() -> Unit) {
+    val fragmentTransaction = beginTransaction()
+    fragmentTransaction.func()
+    fragmentTransaction.commit()
```

Comment:
có thể viết gọn thành 1 
```suggestion
     beginTransaction().apply {
         func()
     }.commit()
```